### ISSUE TITLE
White focus outline missing on button components in Firefox and Safari

### ISSUE THREAD

**Issue by SharonOBoyle** [carbon-design-system/ibm-security#782]
## Bug - 

**Expected behavior -**

When user clicks on some button components (https://ibm-security.carbondesignsystem.com/?path=/story/components-iconbutton--default or https://ibm-security.carbondesignsystem.com/?path=/story/components-button--default) a white outline appears around it to show it's in focus

<img width="122" alt="image" src="https://user-images.githubusercontent.com/39183672/95571367-3affb580-0a20-11eb-96dc-7da9318fae2e.png">

**Actual behavior -**

White outline is missing in some browsers

<img width="114" alt="image" src="https://user-images.githubusercontent.com/39183672/95571466-6e424480-0a20-11eb-8623-228789a0f77d.png">

### Affected browsers

Firefox 81 on OS X 10.15.6
Safari 13 on OS X 10.15.6

### Optional information

Edge & Chrome browsers not affected

**Comment by SimonFinney** [comment]
I suspect this is similar behaviour to https://github.com/carbon-design-system/ibm-security/issues/749#issuecomment-699873958

**Comment by SharonOBoyle** [comment]
Agreed @SimonFinney this due to the browser with OSX 10.x issues described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
